spec/features/visitor_information_page_spec.rb

```diff
@@ -52,6 +52,30 @@
           end
         end
       end
+
+      context "and they are defined by age" do
+
+        it "indicates they are over the specified age for a seat" do
+          enter_visitor_information(flow)
+
+          select '1', from: 'visit[visitor][][number_of_adults]'
+          enter_additional_visitor_information(1, :adult)
+          fill_in "Your first name", with: 'Maggie'
+
+          expect(page).to have_tag('.AgeLabel', :text => 'Over 18')
+        end
+
+        it "indicates they are under the specified age for a seat" do
+          enter_visitor_information(flow)
+
+          select '1', from: 'visit[visitor][][number_of_adults]'
+          enter_additional_visitor_information(1, :child)
+          fill_in "Your first name", with: 'Maggie'
+
+          expect(page).to have_tag('.AgeLabel', :text => 'Under 18')
```

Comment:
I don't think there's any value in that, as in this context there's no concept of variable ages.